prime_numbers.py

```diff
@@ -0,0 +1,31 @@
+def generate_primes(n=100):
+    """
+    This function generates all prime numbers between 1 to n.
+    A prime number is a natural number greater than 1 that has no positive divisors other than 1 and itself.
+    The numbers 0 and 1 are not prime numbers.
+
```

Comment:
Consider using a generator function instead of appending to a list for better memory efficiency. This will be particularly useful when generating a large number of prime numbers.